mm2src/mm2_net/src/grpc_web.rs

```diff
@@ -92,16 +93,31 @@ where
     Ok(msg)
 }
 
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error)]
 pub enum PostGrpcWebErr {
     DecodeBody(String),
     EncodeBody(String),
     InvalidRequest(String),
     Internal(String),
     PayloadTooShort(String),
+    Status(String),
     Transport { uri: String, error: String },
 }
 
+impl Display for PostGrpcWebErr {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        match self {
+            PostGrpcWebErr::DecodeBody(msg) => write!(f, "DecodeBody: {msg}"),
+            PostGrpcWebErr::EncodeBody(msg) => write!(f, "EncodeBody: {msg}"),
+            PostGrpcWebErr::InvalidRequest(msg) => write!(f, "InvalidRequest: {msg}"),
+            PostGrpcWebErr::Internal(msg) => write!(f, "Internal: {msg}"),
+            PostGrpcWebErr::PayloadTooShort(msg) => write!(f, "PayloadTooShort: {msg}"),
+            PostGrpcWebErr::Status(error) => write!(f, "Status: {error:?}"),
+            PostGrpcWebErr::Transport { uri, error } => write!(f, "Transport: URI: {uri}, Error: {error}"),
+        }
+    }
+}
+
```

Comment:
`derive_more` is already added `mm2_net` deps. Please use `derive(Display)` instead.

Reply:
done